src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource/config.properties

```diff
@@ -1,4 +1,5 @@
 bitbucket.scm.credentials=Credentials (for build auth)
```

Comment:
In the other config.properties you remove the "(for build auth)" I think we should be consistent, one or the other, but not both :smile:

Reply:
Think we missed this, should all have the parenthesis.